index.html

```diff
@@ -180,6 +186,46 @@ <h3>Skills</h3>
     </section>
 
   </section>
+
+  <section class="contact-page">
+
+    <h2 class="contact-comment">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+  
+    <form action="https://formspree.io/f/mqkoaqdk" method="post">
+      <ul class="form-list">
+        <li>
+          <label for="form-name"></label>
+          <input type="text" id="form-name" name="user_name" maxlength = "30" required placeholder="Full Name" />
+        </li>
+        <li>
+          <label for="form-mail"></label>
+          <input type="email" id="form-mail" name="user_email" required placeholder="Email address" />
+        </li>
```

Comment:
- [ ] [OPTIONAL] I think it would be great if you add the border on the inputs as pointed out by the previous reviewer. One of the objectives of this project is to understand how to parse a design template precisely. 😎️